app/src/main/res/values/strings.xml

```diff
@@ -114,58 +111,54 @@ Reminder: Easer is still in development. It may contain bugs and the UI definite
     <string name="pref_category_personalise">Personalise</string>
     <string name="pref_category_data_manage">Manage data</string>
 
-    <string name="pref_autostart_title">Autostart on boot</string>
-    <string name="pref_autostart_summary">Start Easer background service automatically</string>
+    <string name="pref_autostart_title">Autostart</string>
+    <string name="pref_autostart_summary">Starts Easer in the background when the device is turned on.</string>
 
     <string name="pref_restart_after_upgrade_title">Start service after upgrade</string>
 
     <string name="pref_welcome_title">Show welcome message</string>
     <string name="pref_welcome_summary">Show welcome message on app startup</string>
 
     <string name="pref_logging_title">Enable external logging</string>
-    <string name="pref_logging_summary">Save logs to external storage devices (/sdcard/logger) to aid bug reports.\nChange of this setting will take effect from next time Easer starts.</string>
+    <string name="pref_logging_summary">Logs to external storage devices (/sdcard/logger) to aid bug reports.\nChanges take effect at next app start.</string>
 
     <string name="pref_use_root_title">Use root features</string>
-    <string name="pref_use_root_summary">Use root permission for specific functions (plugins), which may help resolve compatibility issue on some devices. Root permission won\'t be used elsewhere.</string>
+    <string name="pref_use_root_summary">Use root permission for specific functions (plugins) to help resolve compatibility issues on some devices.</string>
 
-    <string name="pref_cooldown_title">Cool down time</string>
-    <string name="pref_cooldown_summary">Minimal interval between two triggers of the same event</string>
-    <string name="cooldown_time_illformed">The number must not be lower than 0 and not too large to fit in a integer</string>
+    <string name="pref_cooldown_title">Event only once</string>
```

Comment:
I wouldn't say I'm an expert in English, but shouldn't cool-down be easier to understand?

Reply:
I had me confused here, think I forgot the word "per".